Readme.md

```diff
@@ -51,16 +51,16 @@ The *Core Contracts* describe how every DAO generated in the Aragon OSx protocol
 
 In a nutshell, each DAO is composed of 3 interconnecting components:
 
-1. **The DAO treasury:** The DAO contract is where the **core functionality** of the DAO lies. It is the contract in charge of:
+1. **The DAO contract:** The DAO contract is where the **core functionality** of the DAO lies. It is the contract in charge of:
     - Representing the identity of the DAO (ENS name, logo, description, other metadata)
-    - Holding and managing the treasury
+    - Holding and managing the treasury assets
     - Executing arbitrary actions to:
         - transfer assets
         - call its own functions
         - call functions in external contracts
-    - Providing general technical utilities (signature validation, callback handling)
-2. **The Permission Manager:** The permission manager is an integral part of any DAO and the center of our protocol architecture. It **manages permissions for the DAO** by specifying which addresses have permission to call distinct functions on contracts associated with your DAO.
-3. **Plugin Manager**: Any custom functionality can be added or removed through plugins, allowing you to **fully customize your DAO**. Some examples of plugins DAOs could install are:
+    - Providing general technical utilities like callback handling and others
```

Comment:
We may expand a bit here. We have the DAO metadata, the DAO URI (for DAO Star One backend-driven data), we have signature validation, etc